### ISSUE TITLE
Use tag instead of commit for X-AppImage-Version

### ISSUE THREAD

**Issue by Petemir** [kdheepak/taskwarrior-tui#471]
Hello! I am trying to build an automated way of downloading AppImages. This would require checking my current and the latest (at GitHub) versions to see if they differ. If I execute `taskwarrior-tui --version`, I get e.g. 0.23.6. Yet the `X-AppImage-Version` attribute of the AppImage file is 9c4fb86 (i.e. the first 7 chars of the commit hash). I found that most of the applications use the actual version number (e.g. 0.23.6; usually coinciding with the release's tag label) as `X-AppImage-Version`. In fact, AppImage/AppImageKit#672 recommends the following:

```It is recommended to use either a monotonically increasing version number [...] or a form of semantic versioning -- if there is no such information available, a build date might suffice as well).```

Would it be possible to adapt TaskWarrior's CD/CI to use the number? Thanks! :)

**Comment by kdheepak** [comment]
This feels like a bug to me. Did you download the appimage from a tagged release? 

In any case I’ll be traveling for a few weeks and will only be able to look at it then when I’m back at a computer and my desk. Hopefully that is okay. 

Cool work about automating downloading AppImages!

**Comment by Petemir** [comment]
Yes, I always download it from [here](https://github.com/kdheepak/taskwarrior-tui/releases/latest). 
<img width="562" alt="image" src="https://user-images.githubusercontent.com/287105/191501751-668f048d-32a3-4ee1-beee-b0ee999444a1.png">

You can see in the image that the filename is the commit string instead of the tag label. I wasn't sure if it was a bug or a decision,  that's why I'd rather chose the automation tag :). This string is also what appears on the X-AppImage-Version attribute.

Sure, there's no hurry with this, as I won't need to update I shouldn't find any problem ;). Thank you for taking the time to comment in the issue!

re. the automation: well, it's just a basic bash script :) ... I scrap the releases page with [huginn](https://github.com/huginn/huginn/) and when there's a new release I get a telegram message. Then, I run my script which checks my version w.r.t. to the online one and if they are different (sanity check I guess compared to the Telegram notification) it downloads and installs the new AppImage. [AppImageUpdate](https://github.com/AppImage/AppImageUpdate) could be an option too, but well, not everyone will want to support yet another thing that may break in the future :D.

**Comment by kdheepak** [comment]
I’m on a rather slow internet connection so I won’t be able to check this. I’m curious if something changed along the way. I could be remembering incorrectly but I thought then I first implemented this everything had release numbers in the name. In any case I’m sure it’s just a one line fix in the cd GitHub actions, but I’ll only be able to do it later.
